Document guild prune v2 (#571)
discord-userdoccers/discord-userdoccers: pages/gateway/gateway-events.mdx

| guild_id     | snowflake                                                              | The ID of the guild              |
| entitlements | array[[entitlement](/resources/entitlement#entitlement-object) object] | The deleted powerup entitlements |

#### Guild Prune Update

Sent when a guild was pruned or the server computed estimated amount of pruned members.

###### Guild Prune Update Structure

| Field    | Type                                         | Description                         |
| -------- | -------------------------------------------- | ----------------------------------- |
| guild_id | snowflake                                    | The ID of the guild                 |

> | prune    | [guild prune](#guild-prune-structure) object | The information about prune process |

Reviewer: ```suggestion
| prune    | [guild prune](#guild-prune-structure) object | The information about the prune process |
```

###### Guild Prune Structure

| Field         | Type             | Description                          |
| ------------- | ---------------- | ------------------------------------ |
| prune_count   | integer          | Number of pruned members             |
| is_preview    | boolean          | Whether the event is a preview       |
| is_finished   | boolean          | Whether the event finished           |

> | include_roles | array[snowflake] | Additional roles to include          |

Reviewer: ```suggestion
| include_roles | array[snowflake] | Additional roles included          |
```

> | days          | integer          | Number of inactive days to prune for |

Reviewer: ```suggestion
| days          | integer          | Number of inactive days pruned for |
```

| guild_id     | snowflake                                                              | The ID of the guild              |
| entitlements | array[[entitlement](/resources/entitlement#entitlement-object) object] | The deleted powerup entitlements |

> #### Guild Prune Update

Reviewer 2: Can we include the actual event name? I'm presuming in this case that'd be `GUILD_PRUNE_UPDATE`?

Reviewer: this is explained earlier in the page, but all event names are deconstructed to title case for readability, so yes, it would be `GUILD_PRUNE_UPDATE`
